### PR TITLE
New hero banner

### PR DESCRIPTION
**Pull request by Zotman03**
Screen Shot 2022-11-01 at 10 43 25
Screen Shot 2022-11-01 at 10 43 51

Changed the hero-banner page for both teacher and facilitators. Both changed fonts, changed texts, changed colors.

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -11619,6 +11619,16 @@ footer a:hover, footer a:active, footer a:focus {
   padding-top: 9.5rem;
   padding-bottom: 5rem;
 }
+.masthead-teacher{
+  background-color: #FFBA5B;
```
Comment (jennylihan) [inline css/styles.css]: @Zotman03 can you use the variable colors instead of hard-coding the background colors here?

File: css/styles.css [hunk 2]
```diff
@@ -11619,6 +11619,16 @@ footer a:hover, footer a:active, footer a:focus {
...
+  padding-top: 9.5rem;
+  padding-bottom: 5rem;
+}
+.masthead-facil{
+  background-color: #604E6E;
```
Comment (jennylihan) [inline css/styles.css]: here as well!

**Review by jennylihan — CHANGES_REQUESTED** [review]
Looks great otherwise! Please fix the css issue

File: css/styles.css (hunk 1 above)
Comment (Zotman03) [reply css/styles.css]: Ok

File: css/styles.css (hunk 2 above)
Comment (Zotman03) [reply css/styles.css]: ok

**Review by jennylihan — APPROVED** [review]

**Pull request merged**